server/api-util/sdk.js

```diff
@@ -0,0 +1,122 @@
+const http = require('http');
+const https = require('https');
+const Decimal = require('decimal.js');
+const sharetribeSdk = require('sharetribe-flex-sdk');
+
+const CLIENT_ID = process.env.REACT_APP_SHARETRIBE_SDK_CLIENT_ID;
+const CLIENT_SECRET = process.env.SHARETRIBE_SDK_CLIENT_SECRET;
+const USING_SSL = process.env.REACT_APP_SHARETRIBE_USING_SSL === 'true';
+const TRANSIT_VERBOSE = process.env.REACT_APP_SHARETRIBE_SDK_TRANSIT_VERBOSE === 'true';
+
+const typeHandlers = [
```

Comment:
Comment: I was thinking if it's possible/sensible to reuse the type handler code? I guess this is now copy pasted in couple of places.